src/main/NormalFloor.java

```diff
@@ -19,18 +19,29 @@ public NormalFloor(int x, int y, Cell item) {
 	}
 	
 	public boolean interact(Player player, Direction dir){
-		if(box != null){
-			player.setBox(box);
-			box = null;
-			this.setImage('.');
-			return true;
-		}
 		if(zpm != null){
 			player.pickUpZpm();
 			zpm = null;
 			this.setImage('.');
 		}
 		return true;
 	}
+	
+	public Box getBox(){
```

Comment:
should name it like pickUpBox, or something like that,
getBox looks like a getter, which makes no logic, just returns the private object.
it can be confusing later